### PR TITLE
showing messages to user

### PR DESCRIPTION
**Pull request by vikispeaks**

### CODE REVIEW

File: user-authentication/showing-messages-to-user/README.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+It should say that the login attempt was invalid, so users are not left wondering what's going on.
+
+That's what we'll do in this lesson, using **Node.js** `connect-flash` module.
+
+## `connect-flash` Module
+
+When a user is redirected to a certain page, pop messages can be displayed or rendered using the connect-flash module in Node.js.
+
+In our application, we can show messages when users to add or remove items from a to-do list. We can also use it to alert the user every time they log in or out, along with showing any error messages.
```
Comment (mjzac) [inline user-authentication/showing-messages-to-user/README.md:13]:
```suggestion
In our application, we can show messages when users add or remove items from a to-do list. We can also use it to alert the user every time they log in or out, along with showing any error messages.
```

File: user-authentication/showing-messages-to-user/README.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+## Using `connect-flash` to display Flash Messages
+
+To set up `connect-flash` run the below command inside out project folder. As connect-flash requires express, ejs and express-session to work, do make sure all these are installed.
+
+```
+npm i connect-flash --save
+```
+
+This command adds `connect-flash` module to your application and you can start using this on your `index.js`
```
Comment (mjzac) [inline user-authentication/showing-messages-to-user/README.md:23]:
```suggestion
This command adds `connect-flash` module to your application and you can start using this in your `index.js`
```

File: user-authentication/showing-messages-to-user/README.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+```
+
+Make sure your views folder is accessible globally with the below code.
+
+```js
+const path = require("path");
+app.set("views", path.join(__dirname, "views"));
+```
+
+Next let us create a session function to be used within the app using the below code.
```
Comment (mjzac) [inline user-authentication/showing-messages-to-user/README.md:38]: The student will already have set up session from previous lessons. See https://github.com/pupilfirst/wd201-scripts/blob/v2/user-authentication/authentication-using-passport/README.md#now-lets-start-with-the-implementation

**Review by mjzac — CHANGES_REQUESTED** [review]

File: user-authentication/showing-messages-to-user/README.md (hunk 3 above)
Comment (vikispeaks) [reply user-authentication/showing-messages-to-user/README.md:38]: Agree. This will just ask the user to update the same function with addional params.

**Pull request merged**